bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -137,7 +137,7 @@ default String getFieldType() {
 
     @JsonIgnore
     @Override
-    default boolean isEnabled() {
```

Comment:
pls change it back to boolean. we are already past code freeze date. a major version bump in pkg would break far functionalities.

Reply:
removed semantic version change from af-core, and reduced version to 4.3